### ISSUE TITLE
[Feature] Clipboard for RDP connections

### ISSUE THREAD

**Issue by pscriptos** [gnmyt/Nexterm#51]
### General

- [X] My feature does not exist in the latest version of Nexterm.
- [X] I have checked that my feature has not been suggested by anyone else.

### Your Idea

It would be great to have a way to share the clipboard with the RDP connection. 

Either natively, so that I select a text on my host and paste it into the RDP connection, or with a separate button. Then you could press the button, paste the content into a prompt and then “send” it. This text would then be inserted.

**Comment by gnmyt** [comment]
I agree with you, thanks for your suggestion. The feature will be implemented in the next update. :)

**Comment by JordyEGNL** [comment]
@gnmyt for me copy-pasting is only possible on SSH sessions and not for RDP sessions to Windows hosts, is that something on my end or is it not working yet on the Release 1.0.3-OPEN-PREVIEW version?

**Comment by NoRge308** [comment]
> [@gnmyt](https://github.com/gnmyt) for me copy-pasting is only possible on SSH sessions and not for RDP sessions to Windows hosts, is that something on my end or is it not working yet on the Release 1.0.3-OPEN-PREVIEW version?

Hello, same issue for me.
